### DISCUSSION TITLE
How to allow (image) nodes inside Paragraph?

### DISCUSSION THREAD

**Discussion by oliviermills** [ueberdosis/tiptap#1529] [General Questions] [answered]
How do I allow images in paragraphs like this `<p><img ..../></p>` ? This gets ignored by the default TipTap paragraph because it doesn't allow images in paragraphs, ie `inline*` only?. How can I enable this? I tried extending Paragraph with `content: inline* image` but that's not allowed because of mixing inline and block content. I tried changing Image to `inline: true` but that didn't work either

**Comment by robertvanhoesel** [comment, marked as answer]
The Image extension is configured to either have all images be "blocks" living between paragraphs (default) or "inline" inside paragraphs. When adding the extension, configure it to use inline like below.

```ts
this.editor = new Editor({
      extensions: [
        Document,
        Paragraph,
        Text,
        Image.configure({ inline: true })
        //...
      ]
      
```

https://next.tiptap.dev/api/nodes/image/
